web/client/utils/StateUtils.js

```diff
@@ -115,17 +106,16 @@ export const persistEpic = (epic, storeName = PERSISTED_STORE_NAME, name = 'root
     ConfigUtils.setConfigProp(storeName + '.' + name, epic$);
     return (...args) =>
         epic$.mergeMap(e => e(...args));
-
 };
 
 /**
  * Returns a stored root epic
  * @param {string} storeName optional store name the epic is used by
  * @param {string} name optional name (if you want to persist more than one epic)
  */
-const fetchEpic = (storeName = PERSISTED_STORE_NAME, name = 'rootEpic') => {
-    return ConfigUtils.getConfigProp(storeName + '.' + name) || {};
-};
+// const fetchEpic = (storeName = PERSISTED_STORE_NAME, name = 'rootEpic') => {
+//     return ConfigUtils.getConfigProp(storeName + '.' + name) || {};
+// };
```

Comment:
Could we remove this including the relative doc?

Reply:
Removed